src/GeneratedHydrator/Configuration.php

```diff
@@ -220,4 +225,20 @@ public function getClassNameInflector()
 
         return $this->classNameInflector;
     }
+
+    /**
+     * @param \PhpParser\NodeVisitor[] $visitors
+     */
+    public function setCustomVisitors(array $customVisitors)
```

Comment:
I'm fine with bumping to 5.6, don't let it block you.
On Sep 25, 2015 11:58 AM, "Sander Marechal" <notifications@github.com>
wrote:

> In src/GeneratedHydrator/Configuration.php
> <https://github.com/Ocramius/GeneratedHydrator/pull/40#discussion_r40407046>
> :
>
> > @@ -220,4 +225,20 @@ public function getClassNameInflector()
> >
> >          return $this->classNameInflector;
> >      }
> > +
> > +    /**
> > +     * @param \PhpParser\NodeVisitor[] $visitors
> > +     */
> > +    public function setCustomVisitors(array $customVisitors)
>
> Splatting isn't available before PHP 5.6 and the minimum required version
> for this library is now PHP 5.4. But I could do a check that all array
> elements are NodeVisitor instances.
>
> As for constructor injection, custom visitors are optional so I don't
> think these belong in the constructor. IMHO only required settings should
> go in the constructor.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Ocramius/GeneratedHydrator/pull/40/files#r40407046>.
>